client/components/tinymce/plugins/media/restrict-size/index.js

```diff
@@ -29,18 +29,30 @@ function resetImageSrc( img, opening, src, attributes, origAttr, origSrc, closin
 	return `${ opening }${ origSrc }${ attributes }${ closing }`;
 }
 
+function getResizedImgUrlFromImgString( img ) {
+	const parsed = deserialize( img );
+	if ( parsed.media.transient || ! parsed.media.ID ) {
+		return null;
+	}
+
+	return {
+		originalURL: parsed.media.URL,
```

Comment:
Minor: Camel-case guidelines apply to acronyms / abbreviations, i.e. `originalUrl`, not `originalURL`

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions

This unfortunately wouldn't extend to the media object itself since it's referencing properties from the REST API response directly.